cypress/support/fragments/orders/orderLines.js

```diff
@@ -765,10 +783,10 @@ export default {
 
   fillPOLWithTitleLookUp:() => {
     cy.do([
-      orderFormatSelect.choose('Other'),
+      orderFormatSelect.choose(ORDER_FORMAT_NAMES.OTHER),
       acquisitionMethodButton.click(),
-      SelectionOption('Depository').click(),
-      receivingWorkflowSelect.choose('Synchronized order and receipt quantity'),
+      SelectionOption(ACQUISITION_METHOD_NAMES.DEPOSITORY).click(),
+      receivingWorkflowSelect.choose(RECEIVING_WORKFLOW_NAMES.SYNCHRONIZED_ORDER_AND_RECEIPT_QUANTITY),
       physicalUnitPriceTextField.fillIn(physicalUnitPrice),
       quantityPhysicalTextField.fillIn(quantityPhysical),
       materialTypeSelect.choose('book'),
```

Comment:
the same here 
Please use here the variable from the constants file from the support directory